### DISCUSSION TITLE
If any body tried this on Synology's Docker environment, please kindly share on how-to.

### DISCUSSION THREAD

**Discussion by kennethpiu** [kastldratza/zoomrec#56] [Q&A]
I am not really good in docker environment, however, after trying 24 hours to get it running on Synology, it is still no runable, the details are shown below;

1. Plain download code and run on docker
DISPLAY = :1
VNC_COL_DEPTH = 24
VNC_RESOLUTION = 1024x576
VNC_IP = 172.17.0.3
VNC_PORT = 5901
Connect to 172.17.0.3:5901
Start script..
Failed to connect to session manager: Failed to connect to the session manager: SESSION_MANAGER environment variable not defined
mkdir: cannot create directory '/home/zoomrec/.vnc': Permission denied
mkdir: cannot create directory '/home/zoomrec/.vnc': Permission denied
mkdir: cannot create directory '/home/zoomrec/.vnc': Permission denied
mkdir: cannot create directory '/home/zoomrec/.vnc': Permission denied

2. Tried to add mount volume via docker GUI and add/remove puid 1026/1024 pgid 101/100
Still facing mkdir: cannot create directory '/home/zoomrec/.vnc': Permission denied

Hope anybody who is familiar with synology can give some advise. thanks

<img width="649" alt="image" src="https://github.com/kastldratza/zoomrec/assets/36800815/70a02c46-cad9-4a07-96ab-e755341fc566">
<img width="648" alt="image" src="https://github.com/kastldratza/zoomrec/assets/36800815/3c0865ed-e1f9-4b53-bfa0-3aff7cf51670">
<img width="391" alt="image" src="https://github.com/kastldratza/zoomrec/assets/36800815/85f07cd5-3bab-4d4d-97aa-0710a373b760">

Also tried to create new user and group called "Docker" and extract the PUID & PGID which exclusively given permission to access any folders under /Volume/Docker. Still no luck. 
Ref: https://www.reddit.com/r/synology/comments/wf4v27/docker_container_permission_issue/

**Comment by kennethpiu** [comment]
Update: I think I have figure out why there was permission issues, it was due to I have pre-created the zoomrec folder. 
however, now I have encountered another one

Failed to connect to session manager: Failed to connect to the session manager: SESSION_MANAGER environment variable not defined 

>.< help

**Reply by elgroselj** [reply]
I have the same issue.
